Sources/PackageSigning/SigningEntity/SigningEntity.swift

```diff
@@ -40,6 +40,27 @@ public enum SigningEntity: Hashable, Codable, CustomStringConvertible {
         }
     }
 
+    public static func == (lhs: SigningEntity, rhs: SigningEntity) -> Bool {
```

Comment:
we could also consider using something else than == for this check

Reply:
thought about it, but i think it would add complexity to the code and many places to go wrong (there are many places in the code where `SigningEntity` is used as keys in dictionary or elements in set).